decompiler/frontend/binaryninja/handlers/globals.py

```diff
@@ -178,7 +173,7 @@ def _lift_array_type(self, variable: DataVariable, parent: Optional[MediumLevelI
             ssa_label=parent.ssa_memory_version if parent else 0,
         )
 
-    def _lift_basic_type(self, variable: DataVariable, parent: Optional[MediumLevelILInstruction] = None) -> GlobalVariable:
+    def _lift_basic_type(self, variable: DataVariable, parent: Optional[MediumLevelILInstruction] = None, **_) -> GlobalVariable:
         """Lift basic C type found by BNinja (int, float, char, ...)"""
         # If variable references something in address space, then lift it as a pointer (BNinja 4.0 "Error")
         if [x for x in variable.data_refs_from]:
```

Comment:
`self._lifter.lift` should be provided the callers list, right?

Reply:
In this case actually not. (i think?)
The lifter will be called for same address_to lift_pointer_type.
Afterwards the lift call in pointer type will create/manage a valid callers list

Reply:
Could you please add this explanation as a comment?

Reply:
Sure :)